chapters/planning-data-work.tex

```diff
@@ -303,7 +303,14 @@ \subsection{Choosing software}
 % ----------------------------------------------------------------------------------------------
 \section{Organizing code and data}
 
-Organizing files and folders is not a trivial task.
+We assume you are going to do nearly all of your analytical work through code.
```

Comment:
Suggestion:

"We assume you are going to do nearly all of your analytical work through code.
Though it is possible to use some statistical software through the user interface without writing any code, we strongly advise against it.
Writing code creates a record of every task you performed. 
It also prevents direct interaction with the data files that could lead to non-reproducible steps.
Good code, like a good recipe, allows other people to read and replicate it,
and this functionality is now considered an essential component of any research output.
You may do some exploratory tasks by point-and-click or typing directly into the console,
but anything that is included in a research output
must be coded up in an organized fashion so that you can release
the exact code recipe that goes along with your final results.
Still, organizing code and data into files and folders is not a trivial task."

Reply:
Accepted as suggested